lms/static/sass/multicourse/_courses.scss

```diff
@@ -125,8 +157,214 @@
     color: $white;
     text-shadow: none;
 
+    //STATE: hover, focus
     &:hover, &:focus {
       background: $m-blue-l1;
     }
   }
+
+  .filters {
+    @include clearfix();
+    margin-top: ($baseline/2);
+    border-top: 1px solid $courseware-button-border-color;
+    border-bottom: 1px solid $courseware-button-border-color;
+    width: 100%;
+    height: auto;
+    max-height: ($baseline*10);
+
+    ul {
+      @include padding-left(0);
+      margin: 0;
+      list-style: outside none none;
+    }
+
+    li {
+      @include float(left);
+      @include margin(($baseline/2), $baseline, ($baseline/2), 0);
+      padding: ($baseline/2) ($baseline*0.75);
+      position: relative;
+      background: $courseware-button-border-color;
+      width: auto;
```

Comment:
ordering should be: @include float, @include margin, position, padding, background, width